elements/lisk-p2p/src/p2p.ts

```diff
@@ -559,12 +567,15 @@ export class P2P extends EventEmitter {
 	// Make sure you always share shared peer state to a user
```

Comment:
update the comment to something like `Make sure you always share shared peer state to a user and remove private peers`